### DISCUSSION TITLE
Hash errado nos documentos seguintes - o que faço?

### DISCUSSION THREAD

**Discussion by proeps** [assoft-portugal/SAF-T-AO#216] [Q&A]
Quando vou testar o meu SAFT me deparo que o hash apenas dá erro a partir dos documentos seguintes. Por exemplo: Se criar um documento de cada tipo PP, FT, RG, NC, FR nenhum deles terá problemas com o hash.

Mas  a partir do segundo documento de qualquer tipo gera o erro "Hash errado".

Qual poderá ser a causa do problema? Preciso de ajuda.

**Comment by Albert-Nisten** [comment]
Estou tendo um problema semelhante

**Comment by mt-m2g** [comment]
Carissimos conseguiram encontrar alguma solução para os problemas apresetandos? se sim gostaria de apoio ppois tambem me cnontro com o mesmo problema.

Saudações!
